features/sourcemaps.yml

```diff
@@ -0,0 +1,5 @@
+name: Source maps
```

Comment:
I think we should make this name more specific to the fact that this feature is about the HTTP header. There are multiple ways to load source maps into DevTools, and using the HTTP response header is one of them.

Other ways include:

* By including comments in JS: `//# sourceMappingURL=...`
* Automatically, by using [Edge DevTools' Symbol Server integration](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/publish-source-maps-to-azure).
* Manually in DevTools.

Maybe change the name to "Source map header"?

Reply:
Thanks Patrick, in this case, yes, I think the correct thing would be to make this feature just about the header.